### ISSUE TITLE
Control+Backspace Shortcut Causes Node Deletion in ReactFlow Application

### ISSUE THREAD

**Issue by Cristhianzl** [wbkd/react-flow#3328]
### Describe the Bug

I'm currently developing a ReactFlow application that involves the use of text inputs within nodes. However, I've come across an unexpected behavior related to the Control+Backspace shortcut. When attempting to use Control+Backspace to erase text within a text input, the action seems to propagate to ReactFlow itself, resulting in the deletion of the entire node.

### Steps to Reproduce the Bug or Issue

Launch the ReactFlow application.
Create a node containing a text input element.
Input some text into the text field.
Attempt to erase the text by pressing Control+Backspace.

### Expected behavior

The Control+Backspace shortcut should erase the text within the text input field, without affecting the ReactFlow nodes.

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by moklick** [comment]
😱 what is happening here? we will investigate!

**Comment by Cristhianzl** [comment]
> 😱 what is happening here? we will investigate!

I tried a workaround to stop the event when you hit "Control + Backspace" on input. But I'm thinking there's probably a better way. Hoping everything goes smoothly. Thanks for understanding!

**Comment by Cristhianzl** [comment]
> could you try to add the class name "nokey" to your inputs?

Hi @moklick,
I attempted to include the class in the input as follows: className="nokey", but I didn't observe any changes. Could you please confirm if this is the correct approach to address the error?

**Comment by eamador** [comment]
Hi @moklick,

I have been looking for a fix within `useKeyPress` hook (_packages/core/src/hooks/useKeyPress.ts_) and found out that [changing L57](https://github.com/wbkd/react-flow/blob/6d930d0bf9a0971bc8c1d9d2c91b42416eb3e003/packages/core/src/hooks/useKeyPress.ts#L57) fixes the issue:

from
```ts
if (isMatchingKey(keyCodes, pressedKeys.current, false) {
```

to 
```ts
f (isMatchingKey(keyCodes, pressedKeys.current, false) && !modifierPressed.current) {
```

PS: coming from https://github.com/wbkd/react-flow/issues/3416)

**Comment by VictorVelarde** [comment]
Hello, thx for working on this. Do you have an aprox. date for the 11.9.0 release with the fix?

**Comment by moklick** [comment]
Next week 🚀

**Comment by moklick** [comment]
this is fixed in 11.9.0
